src/plugins/intel_cpu/tests/functional/subgraph_tests/src/any_layout.cpp

```diff
@@ -39,38 +38,21 @@ class AnyLayoutOnInputsAndOutputs : public ::testing::TestWithParam<ov::Shape> {
         std::vector<float> output_data(shape_size);
         std::vector<float> expected_output(shape_size, 3);
 
-        // Create CNNNetwork
-        auto ngraph_function = create_test_function(shape);
-        auto cnn = InferenceEngine::CNNNetwork(ngraph_function);
-
-        // Fill inputs and outputs
-        std::vector<std::string> input_names;
-        std::vector<std::string> out_names;
-        for (const auto& it : cnn.getInputsInfo()) {
-            input_names.push_back(it.first);
-        }
-        for (const auto& it : cnn.getOutputsInfo()) {
-            out_names.push_back(it.first);
-        }
-
-        BlobMap inputBlobs;
-        BlobMap outputBlobs;
-
-        TensorDesc tensorDescInp1(Precision::FP32, shape, Layout::ANY);
```

Comment:
Not sure whether still need this test, just migrate to api 2.0.

Reply:
Let's apply tensor layout via PrePostProcessor (like `ppp.input().tensor().set_layout("...")`)